deepsensor/data/loader.py

```diff
@@ -1161,58 +1269,26 @@ def sample_variable(var, sampling_strat, seed):
                     f"with the `links` attribute if using the 'gapfill' sampling strategy"
                 )
```

Comment:
I noticed a bunch of missing code here. I'm confused because your branch is only 2 commits behind `main` (and these 2 commits are irrelevant). Is there any chance you rejected some changes from `main` when merging?

Reply:
Sorry, I noticed that an commited it already, but I think you were already reviewing in between, should be fixed.

Reply:
Is the below section in this diff the missing section? @nilsleh could you help me understand what's happening to the below functionality please? Has it been moved somewhere, is it no longer required? Thanks.

Reply:
@nilsleh ?

Reply:
Resolved by [f5e4a8a](https://github.com/alan-turing-institute/deepsensor/pull/75/commits/f5e4a8aa4cf36ff7cdd8291c558d8ca70c448178)